shared/rust/src/domain/user.rs

```diff
@@ -285,3 +285,21 @@ pub struct UserColorResponse {
     /// The user's colors.
     pub colors: Vec<rgb::RGBA8>,
 }
+
+/// Request for [`CreateFont`]
+#[derive(Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "backend", derive(Apiv2Schema))]
+#[cfg_attr(feature = "backend", openapi(empty))]
```

Comment:
drop the `openapi(empty)` if it compiles without it please. `openapi(empty)` is essentially a workaround to paperclip not understanding some types.